tests/ci/docker_images/linux-x86/amazonlinux-2_clang-11x_cryptofuzz/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+FROM amazonlinux-2023:clang-15x
+
+SHELL ["/bin/bash", "-c"]
+
+RUN set -ex && \
+    dnf -y upgrade --releasever=latest && dnf install -y \
+    libtool \
+    boost-devel \
+    unzip \
+    llvm \
+    llvm-devel \
+    lld && \
+    dnf clean packages && \
+    dnf clean metadata && \
+    dnf clean all && \
+    rm -rf /tmp/* && \
+    rm -rf /var/cache/dnf
+
+ENV FUZZ_ROOT=${DEPENDENCIES_DIR}
+ENV MODULE_ROOT="${FUZZ_ROOT}/modules"
+
+COPY build_cryptofuzz_modules.sh cryptofuzz_data.zip $FUZZ_ROOT/
```

Comment:
what's cryptofuzz_data.zip?

Reply:
This is an initial corpus which has a single test case for each operation/algorithm.